tests/tests_update.yml

```diff
@@ -7,14 +7,26 @@
       vars:
         crypto_policies_policy: LEGACY
         crypto_policies_reload: false
-
   tasks:
     - name: Verify the facts are correctly set
       block:
         - meta: flush_handlers
-
-        - name: Check the current policy is FUTURE
+        - name: Check the current policy is LEGACY
           assert:
             that:
-              - "crypto_policies_active == 'LEGACY'"
-              - "crypto_policies_reboot_required == true"
+              - crypto_policies_active == 'LEGACY'
+              - crypto_policies_reboot_required == true
+  roles:
```

Comment:
I think this will not work as you are redefining the `roles` key that is already defined on line 5. This is in contrast with the other playbook you wrote, where you used `include_role` inside of task. The same for the `tasks` key below.